app/src/main/java/br/org/cesar/knot_setup_app/fragment/thingRegistered/ThingPresenter.java

```diff
@@ -44,8 +39,7 @@
         this.request = "http://" + ip + ":" + port +"/api/devices";
     }
 
-
-    public void getDeviceList(){
+    public void onResume() {
```

Comment:
Why change the name here?
IMO it makes more sense to have the method called getDevicesList than onResume.

Reply:
It was pointed out that the view, should only tell the presenter that some event happened (e.g. a button was clicked, the view resumed, Bluetooth radio was turned off) and the view should NOT use the imperative voice to tell the presenter to do something (e.g. getSomething, setSomething, doSomething). 
As such, I opted to rename several methods to more event-related names. So for example, in this case, the activity was resuming so it seemed appropriate for the presenter method name to be warned about this particular event, hence, the name.
But given your comment about the presenter being responsible to decouple android code from the rest of the code, I think that maybe the solution would be to rename the method to something not android specific but still event-related (onViewResumed?). I would really like to hear your opinion about this.

Reply:
I'd go with `listDevices`, `populateView`, or something like that. I understand that the point is to tell the presenter that something happened, but the name might be confusing/too attached to android lifecycle. 
My point is along the lines of: 
Imagine if you were to change the view now, from something other than Android. The name `onResume` probably wouldn't make that much sense whereas more explicit like the ones above would.